packages/ncids-css/package.json

```diff
@@ -9,7 +9,8 @@
 		"test:sass": "mocha unit/sass-spec.js",
 		"watch": "NODE_ENV=development nswatch",
 		"lint": "stylelint scss/**/*.scss",
-		"lint:fix": "stylelint scss/**/*.scss --fix"
+		"lint:fix": "stylelint scss/**/*.scss --fix",
+    "prepare": "yarn run lint && yarn run build"
```

Comment:
We do not need to lint here. This should be separate step in our workflow. The prepare is just to build things for the docs site really.